docs/data/data-grid/components/toolbar/GridToolbarButton.js

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { DemoContainer } from '@mui/x-data-grid/internals/demo';
+import { Grid } from '@mui/x-data-grid';
+import PrintIcon from '@mui/icons-material/Print';
+
+export default function GridToolbarButton() {
+  return (
+    <DemoContainer>
+      <Grid.Toolbar.Button>
+        <PrintIcon fontSize="small" /> Print
+      </Grid.Toolbar.Button>
+    </DemoContainer>
```

Comment:
Showing these components in isolation looks nice.
But if I copy-paste this snippet, it won't work because it's missing the trigger component.
I see that `DemoContainer` has API ref context, so we can include the trigger in the demo?